docs/community/videos.md

```diff
@@ -136,7 +136,7 @@ Facebook engineers [Bill Fisher](https://twitter.com/fisherwebdev) and [Jing Che
 ### JavaScript Jabber Podcast
 
 [Pete Hunt](http://www.petehunt.net/) and [Jordan Walke](https://github.com/jordwalke) talked about React in [JavaScript Jabber 73](https://devchat.tv/js-jabber/073-jsj-react-with-pete-hunt-and-jordan-walke).
-<figure><a href="http://javascriptjabber.com/073-jsj-react-with-pete-hunt-and-jordan-walke/#content"><img src="../img/docs/javascript-jabber.png"></a></figure>
+<figure><a aria-label="link to play JavaScript Jabber Podcast episode 73" href="http://javascriptjabber.com/073-jsj-react-with-pete-hunt-and-jordan-walke/#content"><img alt="Pete Hunt and Jordan Walke talk about React in JavaScript Jabber 73" src="../img/docs/javascript-jabber.png"></a></figure>
```

Comment:
Do we really need both `alt` and `aria-label` tags? (Aren't they redundant?)

Reply:
I wondered that - would be curious how they actually get read by a screen-reader. Let me try to check that out.

Reply:
Nice catch - we are fine with just the 'aria-label'. :)